packages/ListBox/stories/examples/Keyboard/index.js

```diff
@@ -0,0 +1,333 @@
+import React from "react";
+import Blocks, { Block } from "storybook/components/Blocks";
+
+import ListBox from "../../../src";
+import CardComponent from "../../../../Card/src";
+
+import * as characters from "../../fixtures/characters";
+import Toast from "../../../../Toast/src";
+
+const Card = ({ children }) => <CardComponent style={{ padding: "16px", width: "280px" }}>{children}</CardComponent>;
+const CheckListItem = ({ children }) => {
+  /* eslint-disable jsx-a11y/label-has-associated-control */
+  return (
+    <div style={{ padding: "8px", display: "flex", position: "relative" }}>
+      <label>
+        <div style={{ position: "absolute", top: "10px", left: "-20px" }}>
+          <input type="checkbox" />
+        </div>
+        <div>{children}</div>
+      </label>
+    </div>
+  );
+  /* eslint-enable jsx-a11y/label-has-associated-control */
+};
+
+const Up = () => (
+  <span aria-label="up key" role="img">
+    ↑
+  </span>
+);
+const Down = () => (
+  <span aria-label="down key" role="img">
+    ↓
+  </span>
+);
+
+export function SimpleStory() {
+  return (
+    <>
+      <h2>Simple Case</h2>
+      <Blocks>
+        <Block>
+          <h3>Single</h3>
+          <Card>
+            <ListBox>{characters.antiHeroes}</ListBox>
+          </Card>
+          <h3>Multi</h3>
+          <Card>
+            <ListBox isMulti>{characters.antiHeroes}</ListBox>
+          </Card>
+        </Block>
+        <Block>
+          <CheckListItem>
+            when trigger has focus and pressing <Up /> or <Down /> the popover menu opens
+          </CheckListItem>
+          <CheckListItem>
+            while the menu is open and there is not previous selection and is single selection, the component should
+            wait for an interaction from the user before selecting something either clicking or <Up />
+            <Down />. <br />
+            In case the ListBox has a multi prop enable, then it should focus on the first element if the component has
```

Comment:
I am happy that the single and multi versions now behave the same way!
This sentence can be chopped though ✂️